cdap-examples/LogAnalysis/src/main/java/co/cask/cdap/examples/loganalysis/ApacheAccessLog.java

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
Since we are taking this class from somewhere else I am thinking to keep it as it us as the changes suggested are nothing major. Let me know if thats fine or not.